### PR TITLE
Fix minor indentation

### PR DESCRIPTION
**Pull request by fayimora**
Just code OCD 🙈 

It doesn't seem to render properly in this PR but if you view the actual file, it's now properly indented

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -72,7 +72,7 @@
     "isparta": "4.0.0",
     "istanbul": "0.4.4",
     "json-loader": "0.5.4",
-	"jsdom": "8.5.0",
+	  "jsdom": "8.5.0",
```
Comment (kwelch) [inline package.json]: it is still off? Or is that just github...

Comment (kwelch) [inline package.json]: Looks like it is just the view, I looked at the raw and it looks good.

Comment (kwelch) [inline package.json]: looks like there is a tab character in there.

Comment (fayimora) [inline package.json]: @kwelch it's Github. View the file :)

Comment (kwelch) [inline package.json]: Looking at the raw it is indented too far. 
https://raw.githubusercontent.com/fayimora/react-slingshot/246df149ce08e70e5267d3d2b60e0be6d72299e3/package.json

**Comment by fayimora** [comment]
@kwelch fixed 👍

**Comment by kwelch** [comment]
Thanks!

**Pull request merged**